doc/getting_started/up2.rst

```diff
@@ -1,206 +1,76 @@
-      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/mmcblk0p3 -p 1 -L "ACRN Hypervisor" \
-        -u "bootloader=\EFI\org.clearlinux\bootloaderx64.efi uart=mmio@0x9141e000"
-      # cp /mnt/EFI/org.clearlinux/mnbootloaderx64.efi /mnt/EFI/org.clearlinux/bootloaderx64_origin.efi
-
-Note that the uart address can be obtained from dmesg (after booting Clearlinux on
-your UP2 board)
-
-.. code-block:: none
-
-      # dmesg | grep dw-apb-uart
-      [2.150689] dw-apb-uart.8: ttyS1 at MMIO 0x91420000 (irq = 4, base_baud = 115200) is a 16550A
-      [2.152072] dw-apb-uart.9: ttyS2 at MMIO 0x9141e000 (irq = 5, base_baud = 115200) is a 16550A
-
-The second one associated with ‘dw-apb-uart.9’ is the one on the 40-pin
-HAT connector.
-
-Copy ‘acrn.conf’ to the directory ‘loader/entries’
-
-.. code-block:: none
-
-      # cp hypervisor/bsp/uefi/clearlinux/acrn.conf /mnt/loader/entries
-
-Modify the acrn.conf like below:
-
-.. code-block:: none
+Software setup
+**************
 
-      title The ACRN Service OS
-      linux /EFI/org.clearlinux/kernel-org.clearlinux.pk414-sos.4.14.23-19
-      options **pci_devices_ignore=(0:18:1)** maxcpus=1 console=tty0
-      console=ttyS0 i915.nuclear_pageflip=1 root=PARTUUID=<UUID of
-      /dev/mmcblk0p3> rw rootwait ignore_loglevel no_timer_check
-      consoleblank=0 i915.tsd_init=7 i915.tsd_delay=2000
-      i915.avail_planes_per_pipe=0x00000F
-      i915.domain_plane_owners=0x011111110000 i915.enable_guc_loading=0
-      i915.enable_guc_submission=0 i915.enable_preemption=1
-      i915.context_priority_mode=2 i915.enable_gvt=1 hvlog=2M@0x1FE00000
-      cma=1024M@0x30000000-0
+Setting up the ACRN hypervisor (and associated components) on the UP2 board is no different than other hardware platforms so please follow the instructions provided in the :ref:`getting_started`.
 
-The UUID of the partition /dev/mmcblk0p3 can be found by
+Few parameters specific to the UP2 board (and which differ from what is referenced in the :ref:`getting_started`) are:
 
-.. code-block:: none
+1. Serial Port settings
+#. Storage block device
+#. ``pci_devices_ignore`` parameter
 
-      # blkid /dev/mmcblk0p3
+You will need to keep these in mind in a few places:
 
+* When mounting the EFI System Partition (ESP)
 
-Note that the Clear Linux system gets updated to the latest status when
-adding new packages with bundle-add. The SOS kernel version above is
-only an example. Look at the "/EFI/org.clearlinux/" under the EFI
-partition and adjust the kernel version.
+  .. code-block:: console
 
-.. code-block:: none
+     # mount /dev/mmcblk0p1 /mnt
 
-      # mount /dev/mmcblk0p1 /mnt
-      # ls /mnt/EFI/org.clearlinux
-      bootloaderx64.efi
-      kernel-org.clearlinux.native.4.15.9-538
-      kernel-org.clearlinux.pk414-sos.4.14.23-19
-      kernel-org.clearlinux.pk414-standard.4.14.23-19
-      loaderx64.efi
+* When adjusting the ``acrn.conf`` file
+  * Change the ``options`` line and set ``pci_devices_ignore=(0:18:1)``
```

Comment:
need a blank line before this nested list

Reply:
Done (and there was another one of those in `index.rst` that I'm assuming you missed (otherwise, I may have messed up something that was correct :)